### PR TITLE
Remove locale table + set the default locale

### PR DESCRIPTION
**Pull request by hendrichiatiket**

### CODE REVIEW

**Comment by hendrichiatiket** [comment]
still on progress

File: ui/src/views/Rule/RuleDetail.js [hunk 1]
```diff
@@ -29,6 +29,8 @@ import HitCounterCard from "../Analytic/HitCounterCard";
 
 import axios from "axios";
 
+import localeJson from "../../jsondata/locale.json";
```
Comment (fahrinh) [inline ui/src/views/Rule/RuleDetail.js]: I think it's better to access those static list of locale using a (utility) function. Not directly access the data.

**Review by fahrinh — CHANGES_REQUESTED** [review]
1. Locale code using '_' instead of '-'
2. Locale dropdown in Simulation page is not yet fixed
3. Please fix backend unit tests.
4. IMHO, default locale needs to be stored in database, not hardcoded. So users can change their preference of default locale. Hardcoded list of locale is fine, but not default locale. Maybe it need to be discussed further in the next huddle.

**Comment by fahrinh** [comment]
4. Default locale is not necessary to be stored in database.
Set default locale globally in :
- Hotstone-UI config
- Hotstone backend config

File: ui/src/views/Rule/RuleDetail.js (hunk 1 above)
Comment (hendrichiatiket) [reply ui/src/views/Rule/RuleDetail.js]: now using function getLocales from hotstone.js

**Pull request merged**